airflow/utils/db.py

```diff
@@ -595,6 +595,14 @@ def create_default_connections(session: Session = NEW_SESSION):
         ),
         session,
     )
+    merge_conn(
+        Connection(
+            conn_id="tabular_default",
+            conn_type="tabular",
+            extra='{"extra__tabular__baseUrl": "https://api.tabulardata.io/ws/v1"}',
```

Comment:
Since the base URL is now `host` under the hood, can you update the default connection and the connection doc when you get a chance?

Reply:
Thanks @josh-fell I forgot about this one! Much appreciated